app/views/ingredients/index.html.erb

```diff
@@ -0,0 +1,25 @@
+<div class="container text-center">
+  <div class="row">
+    <div class="col-md-6">
+      <%= link_to "Add new", new_ingredient_path, class: "btn btn-default", type: "new" %>
+      <table class="table">
+        <thead>
+          <tr>
+            <th scope="col">Name</th>
+            <th scope="col">Unit Type</th>
+          </tr>
+        </thead>
+        <% @ingredient.each do |ingredient| %>
```

Comment:
The stuff in this block should be nested for better readability.

Reply:
This is what happens when you don't write tests for your code. I'm not even sure how you verified that it works because in your controller you assigning `@ingredients` and here you call `@ingredient` so if this code would be actually executed then it would result in `NoMethodError` exception being raised.